scripts/devops_tasks/verify_change_log.py

```diff
@@ -58,7 +60,7 @@ def verify_packages(targeted_packages):
         pkg_name, version, _, _ = parse_setup(package)
 
         # Skip management packages
-        if "-mgmt" in pkg_name or pkg_name in excluded_packages:
+        if "-mgmt" in pkg_name or pkg_name in NON_STANDARD_CHANGE_LOG_PACKAGES:
```

Comment:
@scbedd: I was thinking of adding an exclusion logic in common tasks filtering but it's probably better not to clutter it with more filter types in common tasks and control it here. Please let me know if you think otherwise and handle it in different way.

Reply:
No, I think you're fine here.

Reply:
Let's add a log line saying that we're skipping it though.